rust/src/cliwrap/grubby.rs

```diff
@@ -0,0 +1,8 @@
+use anyhow::Result;
+
+/// Primary entrypoint to running our wrapped `grubby` handling.
+pub(crate) fn main(_argv: &[&str]) -> Result<()> {
+    eprintln!("This system is rpm-ostree based; grubby is not used.
```

Comment:
Yeah, I'll blacklist grubby in RHCOS.

But...the other reason to keep this is we could at some point also choose to ship these without wrapping anything; so that someone typing `grubby` gets something friendlier than "command not found".

That said, there are various downsides to that; might be better as bash aliases instead of real binaries so that it only comes into play when humans are typing.

Reply:
So, keeping this one for now.